pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -79,10 +88,28 @@ mod pallet {
         StakeingSettlementAlreadyRecorded,
         /// Exchange rate is invalid.
         InvalidExchangeRate,
+        /// Era has been pushed before.
+        EraAlreadyPushed,
     }
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
+        /// Set era index. Usually happend when era advanced in relaychain.
+        #[pallet::weight(<T as Config>::WeightInfo::set_era_index())]
+        #[transactional]
+        pub fn set_era_index(
```

Comment:
it's called `trigger_new_era` from the spec

Reply:
Fix it in next commit.